projects/js-packages/publicize-components/src/components/connection-verify/index.js

```diff
@@ -42,37 +41,6 @@ class PublicizeConnectionVerify extends Component {
 		}, 500 );
 	};
 
-	renderRefreshableConnections() {
-		const { failedConnections } = this.props;
-		const refreshableConnections = failedConnections.filter( connection => connection.can_refresh );
-
-		if ( refreshableConnections.length ) {
-			return (
-				<Notice className="jetpack-publicize-notice" isDismissible={ false } status="error">
-					<p>
-						{ __(
-							'Before you hit Publish, please refresh the following connection(s) to make sure we can share your post:',
-							'jetpack'
-						) }
-					</p>
-					{ refreshableConnections.map( connection => (
-						<Button
-							href={ connection.refresh_url }
-							isSmall
-							key={ connection.id }
-							onClick={ this.refreshConnectionClick }
-							title={ connection.refresh_text }
-						>
-							{ connection.refresh_text }
-						</Button>
-					) ) }
-				</Notice>
-			);
-		}
-
-		return null;
-	}
-
 	renderNonRefreshableConnections() {
```

Comment:
I think this might never be used as all connections are being returned as refreshable (as long as the user is an admin). It also does the same as above and renders the message received from the server, which I don't think we want to happen, because we'll end up with two notices about broken connections.

Should we get rid of this too?

Reply:
Hmmm, good point. It's only used [here](https://github.com/Automattic/jetpack/blob/e79d560b070e82388a7ffdd8a8af5934fe6b9628/projects/js-packages/publicize-components/src/components/panel/index.jsx#L66), so, like you say, maybe we could dispatch the request in a `useEffect` instead, and get rid of this component altogether.

I think it is ok to have this return `null`, so we could merge this to make sure it makes the release, and then refactor it.